<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>Useful resources for data analysis - </title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="icon" href="/favicon.png">
  <link rel="canonical" href="/posts/usefulresourcesformicrobiomeanalysis/" />
    
    
    <link rel="stylesheet" href="/css/style.min.79132d2a09f76a84636fcc8bf053daeda796005d2f3bd524e3c0c54776caba2f.css">
    <link rel="stylesheet" href="/assets/css/extended.min.b3aee40d9ed5efa41001cf16173b9da286e4f60840081ae9571e90c3fcdb139d.css">

  
    

  
  <style>
    :root {
      --highlight-color: #c2c1c0;
      --dot-color: #c2c1c0;
    }
  </style>

  
    <meta name="description" content="A list of useful packages, tools, and guides for data analysis"/>
    <meta property="og:title" content="Useful resources for data analysis"/>
    <meta property="og:type" content="website"/>
    <meta property="og:url" content="/posts/usefulresourcesformicrobiomeanalysis/"/>
    
    <meta property="og:description" content="A list of useful packages, tools, and guides for data analysis"/>
    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@gutmicrobiome"/>
    <meta name="twitter:creator" content="@gutmicrobiome"/>
  

  
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&amp;display=swap" rel="stylesheet">
  

  
</head>




<body class='page page-blog-single'>
  <div id="menu-main-mobile" class="menu-main-mobile">
    <ul class="menu">
        
        
            
                <li class="menu-item-home">
                    <a href="/">Home</a>
                </li>
            
        
            
                <li class="menu-item-about">
                    <a href="/about/">About</a>
                </li>
            
        
            
                <li class="menu-item-people">
                    <a href="/people/">People</a>
                </li>
            
        
            
                <li class="menu-item-publications">
                    <a href="/publications/">Publications</a>
                </li>
            
        
            
                <li class="menu-item-posts">
                    <a href="/posts/">Posts</a>
                </li>
            
        
            
                <li class="menu-item-tags">
                    <a href="/tags/">Tags</a>
                </li>
            
        
    </ul>
</div>
  <div id="wrapper" class="wrapper">
    <div class='header'>
  <a class="header-logo" href="/"></a>
  <div class="menu-main">
    <ul>
      
      
      
      
      <li class="menu-item-home ">
        <a href="/">
          
          <span>Home</span>
        </a>
      </li>
      
      
      
      <li class="menu-item-about ">
        <a href="/about/">
          
          <span>About</span>
        </a>
      </li>
      
      
      
      <li class="menu-item-people ">
        <a href="/people/">
          
          <span>People</span>
        </a>
      </li>
      
      
      
      <li class="menu-item-publications ">
        <a href="/publications/">
          
          <span>Publications</span>
        </a>
      </li>
      
      
      
      <li class="menu-item-posts ">
        <a href="/posts/">
          
          <span>Posts</span>
        </a>
      </li>
      
      
      
      <li class="menu-item-tags ">
        <a href="/tags/">
          
          <span>Tags</span>
        </a>
      </li>
      
    </ul>
  </div>
  <div id="toggle-menu-main-mobile" class="hamburger-trigger">
    <button class="hamburger">Menu</button>
  </div>
</div>
    

<style>
  :root {
    --highlight-color: #6AC04C;
    --dot-color: #6AC04C;
  }
</style>

  <div class="blog">
    <div class="intro">
      <h1>Useful resources for data analysis<span class="dot">.</span></h1>
      <div class="author">
  <img width="30" class="author-image" src="/images/home/IMG_1974-01.webp" alt="The Holt Lab" />
  <span class="author-name">The Holt Lab</span>
  <span class="author-divider"></span>
  <span class="author-date">July 8, 2024</span>
</div>
      
    </div>
    <div class="content">
      <div class="link-preview">
  <div class="thumbnail-container">
    <a href="https://benjjneb.github.io/dada2/tutorial.html" target="_blank" class="screenshot-link">
      <img src="/images/posts/images_usefulResourcesForMicrobiomeAnalysis/ssDADA2.webp" alt="Example Title" class="link-thumbnail"/>
    </a>
  </div>
  <div class="link-info">
    <h3><a href="https://benjjneb.github.io/dada2/tutorial.html" target="_blank" class="link-text">Microbiomes: DADA2</a></h3>
    <p>I like to analyse all of my microbiome data directly in R with the DADA2 package. This is an Amplicon Sequence Variant (ASV) approach (meaning reads aren't clustered according to a similarity threshold) and relies on run-specific error models to denoise data and limit overestimations of sequence diversity. The DADA2 tutorial is a great place to start if you're interested in this method. Alternatively, you can implement DADA2 in QIIME2. </p>
  </div>
</div>
<div class="link-preview">
  <div class="thumbnail-container">
    <a href="https://anvio.org" target="_blank" class="screenshot-link">
      <img src="/images/posts/images_usefulResourcesForMicrobiomeAnalysis/ssAnvio.webp" alt="Example Title" class="link-thumbnail"/>
    </a>
  </div>
  <div class="link-info">
    <h3><a href="https://anvio.org" target="_blank" class="link-text">Metagenomes and MAGs: Anvio</a></h3>
    <p>Anvio is an immense resource for analysing metagenomes and metagenome-assembled genomes (MAGs). It has a brilliant graphical interface that can be used to manually refine bins but Anvio also wraps a whole suite of tools for pangenomics, phylogenomics, etc. Their webpages include a number of highly detailed tutorials to help you think about your own data and the visualisations are always superb.</p>
  </div>
</div>
<div class="link-preview">
  <div class="thumbnail-container">
    <a href="https://thebrownlab.github.io/phylofisher-pages/" target="_blank" class="screenshot-link">
      <img src="/images/posts/images_usefulResourcesForMicrobiomeAnalysis/ssPhylofisher.webp" alt="Example Title" class="link-thumbnail"/>
    </a>
  </div>
  <div class="link-info">
    <h3><a href="https://thebrownlab.github.io/phylofisher-pages/" target="_blank" class="link-text">Eukaryotic Phylogenomics: Phylofisher</a></h3>
    <p>Phylofisher is a very handy collection of scripts and tools for eukaryotic phylogenomics (i.e. evolutionary histories inferred from multiple genes/proteins). It involves manually inspecting single-gene trees prior to concatenation. Using their precompiled database, you can generate a set of 240 single-gene trees which include 304 reference eukaryotes. To remove contaminants or identity paralogs in single-gene trees, they also provide a very nice application called Parasorter which makes the more labour intensive part of the process much more manageable.</p>
  </div>
</div>
<div class="link-preview">
  <div class="thumbnail-container">
    <a href="https://astrobiomike.github.io/all_tutorials/" target="_blank" class="screenshot-link">
      <img src="/images/posts/images_usefulResourcesForMicrobiomeAnalysis/ssHBB.webp" alt="Example Title" class="link-thumbnail"/>
    </a>
  </div>
  <div class="link-info">
    <h3><a href="https://astrobiomike.github.io/all_tutorials/" target="_blank" class="link-text">Happy Belly Bioinformatics</a></h3>
    <p>This is an excellent resource from Mike Lee and one I often share with those who are new to bioinformatics. It also includes tutorials for amplicon analysis and genomics using some of the methods above.</p>
  </div>
</div>
<div class="link-preview">
  <div class="thumbnail-container">
    <a href="https://pr2-database.org" target="_blank" class="screenshot-link">
      <img src="/images/posts/images_usefulResourcesForMicrobiomeAnalysis/ssPR2.webp" alt="Example Title" class="link-thumbnail"/>
    </a>
  </div>
  <div class="link-info">
    <h3><a href="https://pr2-database.org" target="_blank" class="link-text">PR2</a></h3>
    <p>The PR2 or Protist Ribosomal Reference database is a curated collection of eukaryotic SSU sequences. It is regularly updated and refined, and is now my go-to for any protist microbiome work. I'd also recommend the pr2 R package listed below. </p>
  </div>
</div>
<h1 style="margin-top: 50px;"> R packages </h1>
<input type="text" id="package-search" placeholder="What are you looking for?">
<div class="package-grid">
  <div class="package-item" data-keywords="data science collection">
    <h3><a href="https://www.tidyverse.org/" target="_blank">tidyverse</a></h3>
    <p>A collection of R packages designed for data science.</p>
  </div>
  <div class="package-item" data-keywords="microbiome census data tools">
    <h3><a href="https://joey711.github.io/phyloseq/" target="_blank">phyloseq</a></h3>
    <p>Tools to import, store, analyse, and graphically display microbiome census data.</p>
  </div>
  <div class="package-item" data-keywords="community ecology diversity analysis ordination">
    <h3><a href="https://github.com/vegandevs/vegan?tab=readme-ov-file" target="_blank">vegan</a></h3>
    <p>Community ecology tools for diversity analysis, ordination, and other functions for ecologists.</p>
  </div>
  <div class="package-item" data-keywords="phylogenetic trees visualization annotation">
    <h3><a href="https://github.com/YuLab-SMU/ggtree" target="_blank">ggtree</a></h3>
    <p>Visualisation and annotation of phylogenetic trees.</p>
  </div>
  <div class="package-item" data-keywords="ggtree extensions additional functionalities tree visualization">
    <h3><a href="https://github.com/YuLab-SMU/ggtreeExtra" target="_blank">ggtreeExtra</a></h3>
    <p>Extensions for ggtree, providing additional functionalities for circular tree visualisation.</p>
  </div>
  <div class="package-item" data-keywords="phylogenetic trees functions plotting manipulating">
    <h3><a href="https://cran.r-project.org/web/packages/ape/index.html" target="_blank">ape</a></h3>
    <p>Functions for reading, writing, plotting, and manipulating phylogenetic trees.</p>
  </div>
  <div class="package-item" data-keywords="microbiome data analysis visualization">
    <h3><a href="https://microbiome.github.io/tutorials/" target="_blank">microbiome</a></h3>
    <p>Analysis and visualisation of microbiome data.</p>
  </div>
  <div class="package-item" data-keywords="network analysis visualization">
    <h3><a href="https://r.igraph.org" target="_blank">igraph</a></h3>
    <p>Network analysis and visualisation.</p>
  </div>
  <div class="package-item" data-keywords="tree visualization annotation">
    <h3><a href="https://github.com/YuLab-SMU/treeio" target="_blank">treeio</a></h3>
    <p>Reading and writing phylogenetic tree in various formats.</p>
  </div>
  <div class="package-item" data-keywords="patchwork visualization">
    <h3><a href="https://patchwork.data-imaginist.com/" target="_blank">patchwork</a></h3>
    <p>Organising plots into multi-panelled figures.</p>
  </div>
  <div class="package-item" data-keywords="DESEQ2 differential expression analysis">
    <h3><a href="https://joey711.github.io/phyloseq-extensions/DESeq2.html" target="_blank">DESEQ2</a></h3>
    <p>Differential abundance/expression analysis.</p>
  </div>
  <div class="package-item" data-keywords="ANCOMBC differential abundance analysis">
    <h3><a href="https://www.bioconductor.org/packages/release/bioc/vignettes/ANCOMBC/inst/doc/ANCOMBC2.html" target="_blank">ANCOMBC</a></h3>
    <p>Differential abundance analysis with bias correction.</p>
  </div>
  <div class="package-item" data-keywords="RColorBrewer colour palettes">
    <h3><a href="https://cran.r-project.org/web/packages/RColorBrewer/index.html" target="_blank">RColorBrewer</a></h3>
    <p>ColorBrewer palettes and extrapolation.</p>
  </div>
  <div class="package-item" data-keywords="prismatic colour manipulation">
    <h3><a href="https://github.com/EmilHvitfeldt/prismatic" target="_blank">prismatic</a></h3>
    <p>Colour manipulation.</p>
  </div>
  <div class="package-item" data-keywords="chroma colour manipulation">
    <h3><a href="https://github.com/jiho/chroma" target="_blank">chroma</a></h3>
    <p>Colour manipulation and Plotting in R.</p>
  </div>
  <div class="package-item" data-keywords="pheatmap heatmaps visualisation">
    <h3><a href="https://cran.r-project.org/web/packages/pheatmap/index.html" target="_blank">pheatmap</a></h3>
    <p>Pretty heatmaps.</p>
  </div>
  <div class="package-item" data-keywords="SpiecEasi sparse network inference">
    <h3><a href="https://github.com/zdk123/SpiecEasi" target="_blank">SpiecEasi</a></h3>
    <p>SParse InversE Covariance estimation for Ecological Association and Statistical Inference.</p>
  </div>
  <div class="package-item" data-keywords="ggridges ridge plots visualisation">
    <h3><a href="https://cran.r-project.org/web/packages/ggridges/vignettes/introduction.html" target="_blank">ggridges</a></h3>
    <p>Ridgeline Plots in ggplot2.</p>
  </div>
  <div class="package-item" data-keywords="Biostrings biological sequences analysis">
    <h3><a href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" target="_blank">Biostrings</a></h3>
    <p>Efficient manipulation of biological strings.</p>
  </div>
  <div class="package-item" data-keywords="PR2 database eukaryotic taxonomy phylogeny">
    <h3><a href="https://pr2database.github.io/pr2database/articles/pr2database.html" target="_blank">pr2</a></h3>
    <p>PR2 database interface for eukaryotic taxonomy.</p>
  </div>
</div>
    </div>
  </div>

    <div class="footer">
  <div class="footer-logo">
    <img src="/images/home/HoltLabLogoAllwText_cut.webp" alt="Logo">
  </div>
  <div class="footer-text">
    Made with <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a> and the <a href="https://github.com/zerostaticthemes/hugo-winston-theme" target="_blank" rel="noopener">hugo-winston-theme</a>
  </div>
</div>
  </div>

  

  
    <script type="text/javascript" src="/js/bundle.min.b871dcdf52f94a7b875f9471ee6203a7ab9aac66b1b14afd71b02721e69a86ce.js"></script>
  

  
  

  
  
  
    
  

  

  

</body>
</html>